docs/reference/mapping/types/geo-shape.asciidoc

```diff
@@ -323,9 +360,22 @@ POST /example/doc
 // CONSOLE
 // TEST[skip:https://github.com/elastic/elasticsearch/issues/23836]
 
-*IMPORTANT NOTE:* GeoJSON does not mandate a specific order for vertices thus ambiguous
-polygons around the dateline and poles are possible. To alleviate ambiguity
-the Open Geospatial Consortium (OGC)
+The following is an example of a Polygon with a hole in WKT:
+
+[source,js]
+--------------------------------------------------
+POST /example/doc
+{
+    "location" : "POLYGON ((100.0 0.0, 101.0 0.0, 101.0 1.0, 100.0 1.0, 100.0 0.0),
```

Comment:
`gradle :docs:check` fails here, maybe because of the line break. Not sure how to fix though.

Reply:
If the line break is the problem, it's ok to have long lines.  The code blocks on the docs will scroll